src/ReachSets/ContinuousPost/TMJets/reach.jl

```diff
@@ -0,0 +1,128 @@
+#=Copyright (c) 2018-2019: David Sanders and Luis Benet.
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.=#
+
+using TaylorSeries
+using TaylorIntegration:jetcoeffs!
+using TaylorModels: validated_step!,TaylorModelN
+import IntervalArithmetic
+
+function validated_integ(f!, qq0::AbstractArray{T,1}, δq0::IntervalBox{N,T},
+        t0::T, tmax::T, orderQ::Int, orderT::Int, abstol::T;
+        maxsteps::Int=500, parse_eqs::Bool=true,
+        check_property::Function=(t, x)->true) where {N, T<:Real}
+    # Set proper parameters for jet transport
+
+    @assert N == get_numvars()
+    dof = N
+    # if get_order() != orderQ
+    #     set_variables("δ", numvars=dof, order=orderQ)
+    # end
+    # Some variables
+
+    R = IntervalArithmetic.Interval{T}
+    q0 = IntervalBox(qq0)
+    t   = t0 + Taylor1(orderT)
+    tI  = t0 + Taylor1(orderT+1)
+    δq_norm = IntervalBox(IntervalArithmetic.Interval{T}(-1,1),Val(N))
+    q0box = q0 + δq_norm
+
+    # Allocation of vectors
+    # Output
+    tv    = Array{T}(undef, maxsteps+1)
+    xv    = Array{IntervalBox{N,T}}(undef, maxsteps+1)
+    # xTMNv = Array{TaylorModelN{N,T,T}}(undef, dof, maxsteps+1)
```

Comment:
Could you remove the commented code in this file?

Reply:
Yes. I'd like to make that and in general use our guidelines https://github.com/JuliaReach/JuliaReachDevDocs/